src/CommunityToolkit.Maui.Core/Essentials/Badge/BadgeImplementation.net.cs

```diff
@@ -0,0 +1,11 @@
+namespace CommunityToolkit.Maui.ApplicationModel;
+
+/// <inheritdoc />
+public class BadgeImplementation : IBadge
+{
+	/// <inheritdoc />
+	public void SetCount(int count)
+	{
+		throw new NotImplementedException();
```

Comment:
Let's change this to a `NotSupportedException`

```suggestion
		throw new NotSupportedException($"{nameof(IBadge)} requires a platform-specific implementation of .NET");
```